docs/docs/api/06-Configuration/askui-client.md

```diff
@@ -0,0 +1,82 @@
+---
+custom_edit_url: null
+---
+# askui Client
+
+## Table of Contents
+
+  - [Properties](#properties)
+    - [uiControllerUrl](#uiControllerUrl)
+    - [inferenceClientUrl](#inferenceClientUrl)
+    - [annotationLevel](#annotationLevel)
+    - [credentials](#credentials)
+  - [Set Log Level](#set-log-level)  
+  - [Example](#example)  
+## Properties
+
+### uiControllerUrl
+
+• **uiControllerUrl**: `string` - Default: `http://localhost:6769`
+
+This provides the url where the askui UI Controller runs. The Client is also connecting
+to this address.
```

Comment:
I didn't get this part **The Client is also connecting to this address.**

Reply:
the ui control client is also connecting to the url of the ask UI Controller? or did i misunderstand something